src/java/boa/functions/BoaAstIntrinsics.java

```diff
@@ -278,12 +412,26 @@ private static void openIssuesMap() {
 			e.printStackTrace();
 		}
 	}
+	
+	private static void openSpecMap() {
```

Comment:
Take a look at the open*Map() functions above this one.  The pattern has changed a bit, so that we aren't hard coding boa-njt into the path.  Please update this method to mimic the new pattern of the open*Map() methods.